src/haz3lcore/prog/ModelResults.re

```diff
@@ -1,5 +1,12 @@
 open Sexplib.Std;
 
+/* This module is a map storing ModelResult.t
+      values where the key distinguishes between
```

Comment:
- indentation is too high here
- better phrasing: "ModelResults is used to store the results of evaluations requested by the current editor mode, with the key distinguishing these requests. See the SchoolExercise module for an example."